apps/safe-claiming-app/src/widgets/SnapshotWidget.tsx

```diff
@@ -126,8 +128,10 @@ const SnapshotProposals = ({
 )
 
 const SnapshotWidget = () => {
-  const SNAPSHOT_LINK = "https://snapshot.org/#/safe.eth"
-  const FORUM_LINK = "https://forum.safe.global"
+  const { safe } = useSafeAppsSDK()
+  const snapshotSpace = CHAIN_CONSTANTS[safe.chainId]?.DELEGATE_ID
+
+  const SNAPSHOT_LINK = `https://snapshot.org/#/${snapshotSpace}`
```

Comment:
Nit: you have the same logic in `getSafeSnapshot`. I'd move it to a helper function.